woocommerce-services.php

```diff
@@ -1140,8 +1140,7 @@ public function admin_enqueue_scripts() {
 
 			// Use the same version as Jetpack
 			$jetpack_version = defined( 'JETPACK__VERSION' ) ? JETPACK__VERSION : '0';
-			wp_register_style( 'noticons', $this->wpcom_static_url( '/i/noticons/noticons.css' ), array(), $jetpack_version . '-' . gmdate( 'oW' ) );
```

Comment:
`wp-calypso` now uses a SVG for the checkbox checkmarks, so the `noticons` font can finally be removed 🎉